beacon_node/client/src/builder.rs

```diff
@@ -465,24 +448,23 @@ where
                     .ok_or("Finalized block missing from remote, it returned 404")?;
                 let block_root = block.canonical_root();
 
-                debug!(context.log(), "Downloaded finalized block");
+                debug!("Downloaded finalized block");
 
                 let blobs = if block.message().body().has_blobs() {
-                    debug!(context.log(), "Downloading finalized blobs");
+                    debug!("Downloading finalized blobs");
                     if let Some(response) = remote
                         .get_blobs::<E>(BlockId::Root(block_root), None)
                         .await
                         .map_err(|e| format!("Error fetching finalized blobs from remote: {e:?}"))?
                     {
-                        debug!(context.log(), "Downloaded finalized blobs");
+                        debug!("Downloaded finalized blobs");
                         Some(response.data)
                     } else {
                         warn!(
-                            context.log(),
-                            "Checkpoint server is missing blobs";
-                            "block_root" => %block_root,
-                            "hint" => "use a different URL or ask the provider to update",
-                            "impact" => "db will be slightly corrupt until these blobs are pruned",
+                            block_root = %block_root,
```

Comment:
nit: for structured fields where the variable name matches the field name (i.e. `block_root`) you can just list the variable
i.e. in this case
`%block_root` instead of `block_root = %block_root`. tracing ends up displaying the field name as the variable name in this case